docs/source/DeveloperDocumentation/VUDRM-key-provision.md

```diff
@@ -201,6 +221,221 @@ curl -XGET -H "API-KEY: <api-key>" \
 </cpix:CPIX>
 ```
 
+#### Multikey with seperate bitrates based key
+
+Get a CPIX 2.1 document that uses different content keys per bitrate and one for audio.
+If given a single bitrate is given the document will use one key for all bitrates up to the given bitrate, and one key for the given bitrate up.
```

Comment:
Quite a lot of givens - but the first given should be removed.

Reply:
Yep thats my b, didn't re-write the sentence correctly haha